### PR TITLE
feat(cc): Update links and system requirements

### PR DESCRIPTION
**Pull request by mstroppel**

### CODE REVIEW

File: docs/manufacturer/cloud-connector/index.md [hunk 1]
```diff
@@ -21,13 +21,11 @@ The installer will ask you to accept our [legal information](#legal-information-
...
 
-* Windows 7 SP 2 or newer
-* .NET Framework 4.6.2
+- Windows 7 SP 2 or newer
 
 **Linux:**
 
-* .NET Core: .NET Core 3.1 runtime + ASP .NET Core 3.1 runtime
-* Platform specific builds: Standard C/C++ Libraries + OpenSSL, ICU
+- Platform specific builds: Standard C/C++ Libraries + OpenSSL, ICU
```
Comment (mstroppel) [inline docs/manufacturer/cloud-connector/index.md:28]: How did we come up with these requirements.

File: docs/manufacturer/cloud-connector/index.md [hunk 2]
```diff
@@ -10,7 +10,7 @@ To install the tapio CloudConnector on your machine, you can use the .msi setup
 
 The installer will ask you to accept our [legal information](#legal-information-of-the-cloudconnector).
 
-- [.NET Core](https://github.com/tapioone/CloudConnector/releases/latest/download/tapio.cloudconnector.netcoreapp31.tar.gz)
+- [.NET 6.0](https://github.com/tapioone/CloudConnector/releases/latest/download/tapio.cloudconnector.net60.tar.gz)
```
[the remark this answers is not in the document]
Comment (FreyJim) [reply docs/manufacturer/cloud-connector/index.md:13]: I guess it isn't worse to put the technology name into the file name

File: docs/manufacturer/cloud-connector/index.md (hunk 1 above)
Comment (mstroppel) [reply docs/manufacturer/cloud-connector/index.md:28]: Discussed and keeping it.

File: docs/manufacturer/cloud-connector/index.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (FreyJim) [reply docs/manufacturer/cloud-connector/index.md:13]: we could also go with tapio.cloudconnector.any.tar.gz so we stick to the pattern of runtimes like `Linux ARM`, `Linux x64` ...

[the remark this answers is not in the document]
Comment (mstroppel) [reply docs/manufacturer/cloud-connector/index.md:13]: Suggest to keep for now to avoid further rearrangements that were already tested.

**Review by FreyJim — DISMISSED** [review]

File: docs/manufacturer/cloud-connector/index.md [hunk 3]
```diff
@@ -56,8 +54,8 @@ Unzip the archive on your machine and follow the step. You can also find those s
 Before starting the installation make sure to prepare a CloudConnector configuration for the target machine to have all OPC UA server settings in place.
 
 ```console
-user@example:~$ tar xzvf Tapio.CloudConnector.<platform>.tar.gz
-user@example:~$ cd Tapio.CloudConnector.Service.Core
```
Comment (FelixBurkhardAIT) [inline docs/manufacturer/cloud-connector/index.md:60]: würde hier mal Jim kurz fragen ob das noch passt. Rr musste zumindest eine .dll wieder von `Service` zu `Service.Core` umbennen. Denke aber das der Folder mit `Service` passen sollte.

**Review by FelixBurkhardAIT — APPROVED** [review]

File: docs/manufacturer/cloud-connector/index.md [hunk 4]
```diff
@@ -56,8 +54,8 @@ Unzip the archive on your machine and follow the step. You can also find those s
...
+user@example:~$ tar xzvf tapio.cloudConnector.<platform>.tar.gz
```
Comment (FelixBurkhardAIT) [inline docs/manufacturer/cloud-connector/index.md:57]: müsste afaik 'tapio.cloudconnector' sein. Aber check das bitte nochmal mit dem file gegen

File: docs/manufacturer/cloud-connector/index.md (hunk 3 above)
Comment (mstroppel) [reply docs/manufacturer/cloud-connector/index.md:60]: Folder with service is alright. Just double checked it.

File: docs/manufacturer/cloud-connector/index.md (hunk 4 above)
Comment (mstroppel) [reply docs/manufacturer/cloud-connector/index.md]: ah thanks, fixed it.

**Review by FelixBurkhardAIT — APPROVED** [review]

**Pull request merged**